### ISSUE TITLE
attention_recognition_head.py  sample  y_prev = predicted

### ISSUE THREAD

**Issue by DYF-AI** [Pay20Y/SEED#16]
In attention_recognition_head.py   def smape:    y_prev = predicted? I can't find predicted，and why state = torch.zeros(1, batch_size, self.sDim)？
